### ISSUE TITLE
Apps aren't appearing

### ISSUE THREAD

**Issue by h8uthemost** [cbepx-me/Anbernic-H700-RG-xx-StockOS-Modification#21]
Hi there devs,

Thank you for continuing work on this modified stock. I know a lot of people use it and appreciate it. I am having a problem though. I just tried flashing the new 2.0 on a freshly wiped sd card, and for some reason there's' no app in App Center. So I can't enable ssh to use sftp, along with the other app goodies.

Any idea why the Apps  aren't getting copied?

Thanks for any and all help

EDIT: made my post more clear

**Comment by ahmed2m** [comment]
Apps are no longer in RA. It states in the changelog.
They're in the App center now.

**Comment by h8uthemost** [comment]
I'm aware of that. Sorry, I should worded my post differently. When I open up the App Center hit I get the message No Files. On both TF1 and TF2. So I reflashed and same thing. It's weird.

I use ftp to move my roms onto my cards, so not having an option to enable SSH is an issue for me.

**Comment by ManuelAbril** [comment]
> I'm aware of that. Sorry, I should worded my post differently. My bad for typing folder. When I open up the App Center I get the message No Files. On both TF1 and TF2. So I reflashed and same thing. It's weird.
> 

Some problem here. My workaround:

- Install last RG35XXP image.
- Put upgrade image version into APPs and update.

**Comment by cbepx-me** [comment]
I guess there may be some problems during your operation, but the information currently provided is not enough to determine the cause of the problem, so I'm sorry that I can't help you.

**Comment by h8uthemost** [comment]
@ManuelAbril Well I tried what you said and I got a permission denied error when trying to the move the upgrade files into the Apps folder. I was able to move them with the firmware's browser. That made an upgrade file appear in Apps. However when I ran it, it immediately exits out. So I'm not sure what exactly is going.

I'm trying a reflash now but don't have my hopes up. Thank you for trying though.
